CMSIS/NN/Include/arm_nnsupportfunctions.h

```diff
@@ -349,8 +349,14 @@ void arm_nn_mult_q7(
 
 // Macros for shortening quantization functions' names and avoid long lines
 #define MUL_SAT(a, b)  arm_nn_sat_doubling_high_mult((a), (b))
+#define MUL_SAT_MVE(a, b) arm_mve_sat_doubling_high_mult_32x4((a), (b))
```

Comment:
Just a question about naming: before we called that function with arm_nn prefix. Should we keep the prefix here as well and move mve at the end of the string?

Reply:
@gmiodice Good point.. Moving mve to the end - yes. A reason, I was eager to get rid of the _nn is, it has shown to cause some confusion with  ARM NN. How about arm_sat_doubling_high_mult() and arm_sat_doubling_high_mult_mve_32x4() ?